### PR TITLE
Targeted keywords: HTML/CSS/JS

### PR DESCRIPTION
**Pull request by Dusch4593**
### Description

As part of our ongoing work with target keywords.

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/html/concepts/tags/terms/dl/dl.md
Comment (SSwiniarski) [inline content/html/concepts/tags/terms/dl/dl.md:30]: There seems to be some missing words in this sentence.

File: content/html/concepts/tags/terms/dl/dl.md
Comment (SSwiniarski) [inline content/html/concepts/tags/terms/dl/dl.md:38]: I'm wondering if this example would benefit from a more real-world example of what terms and details might actually be used.

File: content/html/concepts/tags/terms/dt/dt.md
Comment (SSwiniarski) [inline content/html/concepts/tags/terms/dt/dt.md:32]: Again, might benefit from using some more real-world terms/details for the example.

File: content/css/concepts/combinators/combinators.md
Comment (SSwiniarski) [inline content/css/concepts/combinators/combinators.md:48]: The rendered output image might be a little easier to follow if there was some separation between the examples, like a `<hr />` or just some extra spacing.

File: content/css/concepts/selectors/selectors.md
Comment (SSwiniarski) [inline content/css/concepts/selectors/selectors.md:73]: "Conversely," doesn't work here since this is not mirroring the prior statement. It's talking about multiple class values in an element, not on a page. This also should mention that class values, unlike id values, can be duplicated on a page with different elements.

File: content/html/concepts/links/links.md
Comment (SSwiniarski) [inline content/html/concepts/links/links.md:69]: Should we mention that `<link>` elements are placed within the `<head>` element of a page?

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
I reviewed this and left a few comments.

File: content/html/concepts/tags/terms/dl/dl.md
Comment (Dusch4593) [reply content/html/concepts/tags/terms/dl/dl.md:30]: I think I was rewriting the previous sentence on line # 28 by accident. 😌 I removed the text on line # 30.

File: content/html/concepts/tags/terms/dl/dl.md
Comment (Dusch4593) [reply content/html/concepts/tags/terms/dl/dl.md:38]: I'm struggling to come up with one besides glossary/dictionary definitions. 😄

**Comment by Dusch4593** [comment]
@SSwiniarski Finished implemented your feedback. Let me know if there's anything left to add, change, or check 😄

**Review by SSwiniarski — APPROVED** [review]
@Dusch4593, LGTM!

**Review by KTom101 — CHANGES_REQUESTED** [review]
@Dusch4593 Here's a second review! I also noticed that in the images, "termites" is spelled "termintes". That will need to be changed. I've also added this as comments within the review too.

**Comment by Dusch4593** [comment]
@KTom101 implemented your feedback 😄

How do things look on your side?

**Review by KTom101 — CHANGES_REQUESTED** [review]
@Dusch4593 I have a few more comments. Also, I was looking at the "rendered-css-combinators.png" image again and noticed a couple of things.

1. I'm finding the phrasing in the last two examples, "Next sibling after heading", slightly confusing would that be the same as saying the next sibling after the first sibling?
2. The last two headings are both titled "Adjacent Sibling" is the last one supposed to be "General Sibling"?

File: content/css/concepts/selectors/selectors.md
Comment (Dusch4593) [reply content/css/concepts/selectors/selectors.md:71]: Would "second sibling after heading" work better?

Comment (KTom101) [reply content/css/concepts/selectors/selectors.md]: @Dusch4593 That works!

**Comment by Dusch4593** [comment]
@KTom101 finished implementing the new changes. Thank you for the feedback 😄

File: content/css/concepts/selectors/selectors.md
Comment (KTom101) [inline content/css/concepts/selectors/selectors.md:18]:
```suggestion
All markups can be selected with the `*` selector:
```

**Review by KTom101 — COMMENTED** [review]
@Dusch4593 Added some more things and I noticed there is a "Codecademy Web Browser" pop-up in the "rendered-css-combinators.png".

Also, perhaps we can have one more reviewer since there are quite a few files.

**Comment by Dusch4593** [comment]
@KTom101 Finished implementing your suggested changes 😄

**Review by KTom101 — APPROVED** [review]
👍

File: content/javascript/concepts/arrays/terms/shift/shift.md
Comment (caupolicandiaz) [inline content/javascript/concepts/arrays/terms/shift/shift.md:54]:
```suggestion
The instructions of the `alfredoRecipe` array are followed by a `while` [loop](https://www.codecademy.com/resources/docs/javascript/loops) that prints each step of the recipe to the console:
```

Comment (caupolicandiaz) [reply content/javascript/concepts/arrays/terms/shift/shift.md:54]: The original felt a bit indirect

File: content/javascript/concepts/arrays/terms/indexOf/indexOf.md
Comment (caupolicandiaz) [inline content/javascript/concepts/arrays/terms/indexOf/indexOf.md:30]:
```suggestion
In the example below, the index for the number 12 is logged to the console:
```

Comment (caupolicandiaz) [reply content/javascript/concepts/arrays/terms/indexOf/indexOf.md:30]: Not sure my suggestion is ideal but the original is confusing to my ear. It suggests to me that the index = 12 as opposed to the index of the value 12 in the list

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@Dusch4593 looks pretty clean just a couple of comments

**Comment by Dusch4593** [comment]
@caupolicandiaz Implemented your suggestions 😄

**Pull request merged**
